### PR TITLE
Internal: media queries template

### PR DESCRIPTION
**Pull request by nitish24p**
This PR creates a template for adding media queries to subsequent styling. The PR follows a mixin approach for adding media queries.

```scss
@mixin respond-to($media) {
}
```

A global function respond to is created and the arguement passed to it is the breakpoint for the media query

eg,

```scss
  .header {
    height: 50px;
    @include respond-to(mobile){
      height: 40px // This will override the height in mobile screen sizes
    }

    @include respond-to(tablet){
      height: 30px // This will override the height in tablet screen sizes
    }
  }
```

### CODE REVIEW

**Comment by nitish24p** [comment]
@crysfel i made the changes.. but why do i have 21 file changes 🤔

**Comment by crysfel** [comment]
@nitish24p It seems like this PR is trying to add commits from other people as well, could you rebase your branch with the latest? I think that might solve the problems.

Take a look at the commits tab, and you will see commits from other people.

**Comment by nitish24p** [comment]
@arecvlohe sorry had to do a force push.. had messed up my branch.. can you plz review these again :)

File: src/theme/index.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import "base/variables";
 @import "base/elements";
+@import "base/mediaquerries";
```
Comment (arecvlohe) [inline src/theme/index.scss]: Also change the spelling here for `querries` to `queries`

**Comment by arecvlohe** [comment]
For anyone interested to know what a querry is because I didn't know either:

screen shot 2018-09-19 at 1 55 39 pm
screen shot 2018-09-19 at 1 55 20 pm

**Comment by linux-nerd** [comment]
@arecvlohe, @nitish24p  Don't you think **mediaqueries.scss** is little difficult to read. May be we can either follow kebab case or camel case for readability. What's your take on this?

**Comment by arecvlohe** [comment]
@linux-nerd I think the convention for css/scss is for files to be lowercase. I found this article which has several examples of lowercase naming: http://thesassway.com/beginner/how-to-structure-a-sass-project. On the flip side, with BEM, they are using underscores: https://en.bem.info/methodology/filestructure/. I am for lowercase because it _seems_ to be a convention that most people are used to. It would be good to hear other opinions on this as well. What do you all think @crysfel and @emmawedekind? Also, anyone else can chime in as well.

**Comment by crysfel** [comment]
@linux-nerd yes, let's stick to lowercase for file names

**Pull request merged**